app/auto_teams.go

```diff
@@ -37,14 +37,14 @@ func NewAutoTeamCreator(client *model.Client4) *AutoTeamCreator {
 	}
 }
 
-func (cfg *AutoTeamCreator) createRandomTeam() (*model.Team, bool) {
+func (cfg *AutoTeamCreator) createRandomTeam() (*model.Team, error) {
 	var teamEmail string
 	var teamDisplayName string
 	var teamName string
 	if cfg.Fuzzy {
 		teamEmail = "success+" + model.NewId() + "simulator.amazonses.com"
 		teamDisplayName = utils.FuzzName()
-		teamName = utils.FuzzName()
+		teamName = utils.RandomName(cfg.NameLength, cfg.NameCharset) + model.NewId()
```

Comment:
out of curiosity, why not FuzzName?

Reply:
Ah, I'd forgotten this change -- `utils.FuzzName` generates decidedly invalid team names, both in length and number of characters. I have no idea how this ever worked.

Reply:
IIRC there's a new random team name utility fn in the server code, since we ran into this last time I was on SET.

Reply:
I found this in the `model` package:
https://github.com/mattermost/mattermost-server/blob/9a531fa0fbb5e8592739c21ba1553a622e71cb73/model/utils.go#L149-L156

but it looks like it's just amounts to `model.NewId()` in the end as well. @cpoile, do you know if there was a "better" one?

Reply:
I think that is the better one -- it generates valid team names (the SET problem was that random team names would sometimes include reserved words).